src/bin.mts

```diff
@@ -45,6 +46,20 @@ const args = yargs(process.argv)
     description: 'Override the VS Code version used to run tests',
     group: vscodeSection,
   })
+  .option('install-extensions', {
+    alias: 'e',
+    type: 'array',
+    description:
+      "A list of vscode extensions to install prior to running the tests. Can be specified as 'owner.extension','owner.extension@2.3.15', 'owner.extension@prerelease', or the path to a vsix file (/path/to/extension.vsix)",
+    group: vscodeSection,
+  })
+  .option('install-extension-dependencies', {
+    alias: 'd',
+    type: 'array',
```

Comment:
This is incorrectly defined as an array. Also, this should probably be default behavior, perhaps instead a flag `--no-extension-dependencies` would be better

Reply:
Yes that was one of my questions whether it should be opt-in or opt-out behavior. I agree opt-out is best if we aren't worried about breaking changes at this point.

Reply:
68c968f